tests/lib/rules/no-extra-parens.js

```diff
@@ -152,7 +154,6 @@ ruleTester.run("no-extra-parens", rule, {
         "var o = { foo: (function() { return bar(); })() };",
         "o.foo = (function(){ return bar(); })();",
         "(function(){ return bar(); })(), (function(){ return bar(); })()",
-        "((function(){ return bar(); })())",
```

Comment:
Because we might have added this exception as a result of an issue. In that case, we need to be sure that we aren't creating a regression.

If this case was there since the rule was created, then it's more obvious to say its a bug we should fix.